addons/sale_expense/__init__.py

```diff
@@ -1,4 +1,3 @@
-# -*- coding: utf-8 -*-
```

Comment:
This just increases probability for conflicts with no benefits, you are not even touching this(these) file(s) otherwise...

Reply:
Easy to solve but annoying because it stops the automatic forward port

The decision was made, and not by me, to not remove these all at once on the whole repo, you are kind of doing that.